tests/jballerina-unit-test/src/test/resources/test-src/jvm/types.bal

```diff
@@ -90,6 +90,14 @@ function testIntToByteExplicitCast(int b) returns byte|error {
     return a;
 }
 
+function testXMLSequenceWithOneTextMember() {
+    xml text1 = xml `<a>abc</a><b>def</b>`;
+    xml text2 = xml:map(xml:elements(text1), y => y.getChildren());
+    xml:Text t1 =  <xml:Text>text2;
+    xml:Text text3 = <xml:Text>xml`abcdef`;
```

Comment:
```suggestion
    xml:Text text3 = xml `abcdef`;
```

Why do we need the cast?

Reply:
Variable names seem to be a mix too.

Reply:
Fixed with dd8e19f2a0e5255dbe863e931bffe66bf0190024